From 5dea212de41d2bd57b8fb70f38b820a99e98a409 Mon Sep 17 00:00:00 2001
From: Wu Liang feng <wulf@rock-chips.com>
Date: Fri, 25 Mar 2016 11:58:43 +0800
Subject: [PATCH] ARM64: dts: rk3399-monkey: set usbdrd_dwc3_0 in peripheral
 mode

Set dwc3_0 in peripheral only mode until Type-C function is
ready, and then we can set dwc3_0 in drd mode.

Change-Id: I0ccb92db97244d7a34dd17c58757fc5aa1b11dac
Signed-off-by: Wu Liang feng <wulf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-monkey.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts b/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
index e352c9945aa9..9ba5aa63494c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
@@ -133,3 +133,7 @@
         status = "okay";
 };
 
+&usbdrd_dwc3_0 {
+	dr_mode = "peripheral";
+};
+
-- 
2.35.3

